client/my-sites/stats/stats-date-control/stats-date-control-picker-shortcuts.tsx

```diff
@@ -8,11 +8,15 @@ const DateControlPickerShortcuts = ( {
 	currentShortcut,
 	onClick,
 }: DateControlPickerShortcutsProps ) => {
+	// Simple selection test.
+	function isSelectedShortcut( shortcut: DateControlPickerShortcut ) {
+		return shortcut.id === currentShortcut;
+	}
 	// Apply selection state via CSS.
 	function classNameForShortcut( shortcut: DateControlPickerShortcut ): string {
 		const defaultClassName = 'date-control-picker-shortcuts__shortcut';
 		const selectedClassName = defaultClassName + ' is-selected';
-		return shortcut.id !== currentShortcut ? defaultClassName : selectedClassName;
+		return isSelectedShortcut( shortcut ) ? selectedClassName : defaultClassName;
```

Comment:
This is not related to the PR and I missed the note before.

We are heavily using lib `classnames` which is super handy for scenarios like this. We could simple do the following in the tag  `<li>`:

```JSX
<li className={classNames('date-control-picker-shortcuts__shortcut', {'is-selected': isSelectedShortcut( shortcut ) })}>
```

Do you think it looks clearer?

Reply:
I'll have another look at this. Thanks for the reminder about the `classnames` call.